api/controllers/game/join-rematch.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Join the two players into the new game
+ */
+module.exports = async function (req, res) {
+  try {
+    const { usr: userId } = req.session;
+
+    const user = await User.findOne({ id: userId });
+    const oldGame = await Game.findOne({ id: user.rematchOldGame });
+
+    const newGameId = oldGame.rematchGame;
+    const game = await gameService.populateGame({ gameId: newGameId });
+    Game.subscribe(req, [game.id]);
+
+    req.session.game = game.id;
+    req.session.pNum = user.pNum;
```

Comment:
I _think_ we could get rid of the `User.rematchOldGame` column by queries for the old game either based on your existing session depending on when that's set, or using a req param. Basically as long as we have the id of one game or the other, (old or new) and as long as we know whether it's the old game or the new game, we could find the other one without data on the user.

If we have the new game id (i.e. the rematch id) and want the old game, we could:
```
const oldGame = await Game.findOne({ rematchOldGame: rematchGameId });
```